web/src/styles/sections/nav-bar.scss

```diff
@@ -6,3 +6,24 @@
 .nav-bar-override-usa-accordion__content {
   padding: 0px 0px 12px 16px !important;
 }
+
+.menu-item-title.Mui-disabled {
+  opacity: 1;
+}
+
+$icon-offset: 45px;
```

Comment:
Should we use rem instead?

Reply:
So their might be a better way to write this if I gave it more thought but the idea was that the offset makes up for an icon not being with the title so as long as the Icon size is in PX we should keep the offset in PX.